doc/comandi/converti-opzioni.md

```diff
@@ -1,54 +1,36 @@
 
 ## Opzioni di `converti`
 
-Elenchiamo subito le opzioni. Dopo l'elenco trovate una spiegazione
-sul come usarle
+Di seguito è possibile trovare l'elenco delle opzioni di 'converti' e una breve descrizione
 
-### `collegamento-normativa`
+### Tutte le opzioni di 'converti' sono  suscettibili di errori di sintassi rST, pertanto i risultati andranno verificati. 
```

Comment:
recepisco la tua idea di riassumere il concetto una volta per tutte. Lo farei diversamente. Qui stai usando una formattazione da header per qualcosa che è più una nota, e potrebbe essere un semplice paragrafo in grassetto. Comunque accetterei questa modifica per poi cambiare di nuovo il file

Reply:
Ho riletto le opzioni ... due di esse possono introdurre errori di sintassi (`collegamento-normattiva`, `dividi-sezioni`), una di esse elimina gli errori (`celle-complesse`) mentre la quarta non ha nessun effetto sugli errori ... quindi non credo che possiamo riassumere gli effetti in una nota simile, manterrei la formulazione corrente